electron/main.ts

```diff
@@ -94,12 +94,31 @@ const tsStore = new Store({
 async function createWindow(): Promise<BrowserWindow> {
   const { exitToTray, startInTray } = await GlobalConfig.get().getSettings()
 
+  let windowProps: Electron.Rectangle = {
+    height: 690,
+    width: 1200,
+    x: 0,
+    y: 0
+  }
+
+  if (store.has('window-props')) {
+    const tmpWindowProps = store.get('window-props') as Electron.Rectangle
+    if (
```

Comment:
```suggestion
    if (
      tmpWindowProps &&
      tmpWindowProps.width &&
      tmpWindowProps.heigh &&
      tmpWindowProps.y !== undefined &&
      tmpWindowProps.x !== undefined
    ) {
      windowProps = tmpWindowProps
    }
```

in Javascript we don't need to check if it's different from `undefined` since `undefined` is a `falsy` value. Unless in some specific situations when the value can be falsy as well. So unless some of those values can be 0, for instance, then you can change this suggestion. I guess perhaps y and x could be zero but not width and height.

Reply:
Imo we can keep it this way for consistency.